cmd/benchmark.go

```diff
@@ -375,3 +400,30 @@ func generateDataStreamCorpusCommandAction(cmd *cobra.Command, _ []string) error
 
 	return nil
 }
+
+func initializeESMetricsClient(ctx context.Context) (*elasticsearch.Client, error) {
+	address := os.Getenv(system.ESMetricstoreHostEnv)
+	user := os.Getenv(system.ESMetricstoreUsernameEnv)
+	pass := os.Getenv(system.ESMetricstorePasswordEnv)
+	cacert := os.Getenv(system.ESMetricstoreCACertificateEnv)
+	if address == "" || user == "" || pass == "" {
+		logger.Debugf("can't initialize metricstore, missing environment configuration")
+		return nil, nil
+	}
+
+	esClient, err := elasticsearch.NewClient(
+		elasticsearch.OptionWithAddress(address),
+		elasticsearch.OptionWithUsername(user),
+		elasticsearch.OptionWithPassword(pass),
+		elasticsearch.OptionWithCertificateAuthority(cacert),
+	)
```

Comment:
I think this client should be created with `stack.NewElasticsearchClient` method instead of the elasticsearch one, so it follows the same approach as in other modules. 
```suggestion
	esClient, err := stack.NewElasticsearchClient(
		elasticsearch.OptionWithAddress(address),
		elasticsearch.OptionWithUsername(user),
		elasticsearch.OptionWithPassword(pass),
		elasticsearch.OptionWithCertificateAuthority(cacert),
	)
```